contributing/content-style-guide.md

```diff
@@ -307,10 +307,10 @@ A numbered list following an H2 heading:
 ## Tables
 Tables provide a great way to present complex information and can help the content be more scannable for users, too.
 
-There are many ways to construct a table, like row spanning and cell splitting. Make sure the content is clear, concise, and presents well on the web page (like avoid awkward word wrapping).
+There are many ways to construct a table, like row spanning and cell splitting. Make sure the content is clear, concise, and presented well on the web page (like avoiding awkward word wrapping).
```

Comment:
We should avoid using "like" to present examples.
```suggestion
There are many ways to construct a table, such as row spanning and cell splitting. The content should be clear, concise, and presented well on the web page (for example, avoid awkward word wrapping).
```

Reply:
i hear you. might be worth discussing with the docs team bc the [style guide](https://github.com/dbt-labs/docs.getdbt.com/blob/2db0204a02fdf9150732662b7369ba50c7a6f65f/contributing/content-style-guide.md#latin-abbreviations) itself suggests using 'like' (and i was also told to use 'like', 'such as', 'for example', as opposed to 'eg' when i started).

![Screenshot 2023-10-27 at 11 48 19](https://github.com/dbt-labs/docs.getdbt.com/assets/89008547/71288181-6d6b-4cf8-a366-c3cd8e459d81)